src/stylesheets/elements/_buttons.scss

```diff
@@ -195,6 +212,14 @@ $button-stroke: inset 0 0 0 units($theme-button-stroke-width);
   }
 }
 
-.usa-button-unstyled {
-  @include button-unstyled;
+.usa-button,
+.usa-button--accent-cool,
+.usa-button--base,
+.usa-button--big,
+.usa-button--inverse,
+.usa-button--outline,
+.usa-button--secondary {
+  &.usa-button--unstyled {
+    @include button-unstyled;
+  }
```

Comment:
Why is this necessary to add the variant button classes before `.usa-button--unstyled`. It looks like it works without this and keeps the CSS much smaller and shallower.

Reply:
It was defensive specificity, but I suppose we could remove it

Reply:
Actually, it does fix some specificity stuff around hover/active/focus

Reply:
I'm not able to reproduce when I check the button states? Is there something else I should do to see the issue?

Reply:
I see one issue with the outline button on hover but can probably be resolved individually by removing the outline.

Reply:
Do you want to resolve [those issues](https://github.com/uswds/uswds/pull/2974#issuecomment-474956088) individually, chain only those classes, or chain all classes (even if most don't need it)?

Reply:
I'm inclined to leave it as is and apply the specificity across the board, if only to make it clear that the `unstyled` variant _should_ always override any other variant styling. But I don't feel super strongly about this.

Reply:
Made a few tweaks to the mixin in #2976 to make it more robust and fix these two issues. If that looks OK, we can merge it in.

Reply:
LGTM